tests/HTMLPurifierTest.hack

```diff
@@ -209,7 +294,9 @@ class HTMLPurifierTest extends HackTest {
 		echo "\nrunning testPolicyAllowListClean()...";
 		//porting over first config classes....
 		$config = HTMLPurifier\HTMLPurifier_Config::createDefault();
-		$config->def->defaults['HTML.Allowed'] = 'b';
+		$policy = HTMLPurifier\HTMLPurifier_Policy::fromEmpty();
+		$policy->addAllowedTag(Enums\HtmlTags::B);
+		$purifier = new HTMLPurifier\HTMLPurifier($config, $policy);
 		$dirty_html = '<b>Hello';
 		$purifier = new HTMLPurifier\HTMLPurifier($config);
```

Comment:
Does this line mean the other `$purifier` gets overwritten or is there some weird singleton behavior that I don't understand 😅

Reply:
Nope - it does get overwritten but since it's overwritten with the default tags (b, p, i) it does the same thing as if the allowed tag list only contained b 😅 Updated this to take out the second one for clarity's sake